jenkins-integration/beaker/README.md

```diff
@@ -43,3 +52,40 @@ When running the PE installer, two options can be set:
   `pe_ver` environment variable
 
 You must at least set the PE dist dir. If `pe_ver` isn't set, it will install the latest version at the location provided by the dist dir.
+
+## gatling/
+### 10_start_gatling_scenario.rb
+This script starts a gatling scenario, which will run against some SUT. It clones this repo to a hopefully unique directory, `gatling-puppet-load-test_<simulation_id>`, where all the gatling output will be stored.
+
+It needs some environment variables to be set:
+* `GATLING_SIMULATION_ID`: An identifier for the run
+* `GATLING_SCENARIO`: A gatling scenario file from the `simulation-runner/config/scenarios/` directory. E.g.: `foss375-catalogzero-100.json`
+* `GATLING_SUT_HOSTNAME`: The hostname of the machine the scenario should be run against
+
+It is intended to be used at the same time as `20_start_memory_watcher.rb`, and and to differentiate between the gatling driver and the SUT, your beaker hosts file might look something like this:
+
+```yaml
+HOSTS:
+  wzmg0128dylvv1d:
+    platform: el-6-x86_64
+    roles:
+    - agent
+    - sut
+  kf58a5eknd06u1s:
+    platform: el-6-x86_64
+    roles:
+    - agent
+    - master
+    - gatling
+```
```

Comment:
You wouldn't really have the master and gatling on the same node usually, would you?